src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
+const ANIMAL_HELTH = 100;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = ANIMAL_HELTH;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide(isHidden = true) {
+    this.hidden = isHidden;
```

Comment:
if the function has a name hide this function should just hide but not toggle
in your case I can call like this `hide(false)`